### PR TITLE
package-not-maintained needs a rework

### PR DESCRIPTION
**Pull request by OniriCorpe**
we need to:

- better warn users (be more explicit about the sensitive reliability of non-maintained packages)
- recruit maintainers (clearly state the need for help)

we also need to tag packages who does not have maintainer in their manifest: https://github.com/YunoHost/issues/issues/2360

### CODE REVIEW

**Comment by OniriCorpe** [comment]
maybe add 
`You can [learn how to package](https://yunohost.org/packaging_apps_intro) if you'd like to adopt this package.`
at the end

**Review by Psycojoker — APPROVED** [review]

**Review by tituspijean — APPROVED** [review]

File: antifeatures.toml [hunk 1]
```diff
@@ -79,11 +79,11 @@ description.it = "Questo software non è più mantenuto. Ci si può aspettare ch
...
-title.fr = "Paquet non maintenu"
+title.fr = "Package non maintenu"
 title.it = "Pacchetto non mantenuto"
-description.en = "This YunoHost package is not maintained and needs adoption."
+description.en = "This YunoHost package is not actively maintained and needs adoption. This means that minimal maintenance is made by volunteers who don't use the app, so you should expect the app to lose reliability over time. You can [learn how to package](https://yunohost.org/packaging_apps_intro) if you'd like to adopt it."
 description.eu = "Pakete honek ez du mantenduko duenik, boluntario baten beharra dauka."
-description.fr = "Ce package YunoHost n'est plus maintenu et doit être adopté."
+description.fr = "Ce package YunoHost n'est pas activement maintenu et a besoin d'être adopté. Cela veut dire que la maintenance minimale est réalisée par des bénévoles qui n'utilisent pas l'application, il faut donc s'attendre à ce que l'app perde en fiabilité avec le temps. Vous pouvez [apprendre comment packager](https://yunohost.org/packaging_apps_intro) si vous voulez l'adopter."
 description.it = "Questo pacchetto di YunoHost non è più mantenuto e necessita di essere adottato."
```
Comment (alexAubin) [inline antifeatures.toml:87]: (watdo about the .it and .eu string tho ? Imho I would remove those since they are completely out of sync)

Comment (OniriCorpe) [reply antifeatures.toml:87]: we could poke xabi (for eu) and tomi (for it) when we're decided on the phrasing?

Comment (OniriCorpe) [reply antifeatures.toml:87]: also @Psycojoker want to put those files on weblate

Comment (OniriCorpe) [reply antifeatures.toml:87]: @xabirequejo @xplosionmind do you want to update the basque and italian translations before merging?

Comment (xabirequejo) [reply antifeatures.toml]: I won’t have access to a reliable computer until 18:00.

File: antifeatures.toml [hunk 2]
```diff
@@ -78,14 +78,15 @@ title.fr = "Application non maintenue "
...
+description.eu = "YunoHost pakete honek ez du mantenduko duenik, bere gain hartuko duen norbaiten beharra dauka. Honek esan nahi duena da mantentze-lanak minimoak izango direla eta aplikazioa erabiltzen ez duten boluntarioek egingo dituztela lanok; denborak aurrera egin ahala fidagarri izateari utziko dio. [Aplikazioak nola paketatu](https://yunohost.org/packaging_apps_intro) ikas dezakezu, zure gain hartu nahi baduzu."
+description.fr = "Ce package YunoHost n'est pas activement maintenu et a besoin d'être adopté. Cela veut dire que la maintenance minimale est réalisée par des bénévoles qui n'utilisent pas l'application, il faut donc s'attendre à ce que l'app perde en fiabilité avec le temps. Vous pouvez [apprendre comment packager](https://yunohost.org/packaging_apps_intro) si vous voulez l'adopter."
 description.it = "Questo pacchetto di YunoHost non è più mantenuto e necessita di essere adottato."
 icon = "user-times"
+icon = "user-times"
```
Comment (alexAubin) [inline antifeatures.toml:86]:
```suggestion
```

**Pull request merged**